test/cases/030_security/010_ports/test.yml

```diff
@@ -0,0 +1,23 @@
+kernel:
+  image: "linuxkit/kernel:4.9.x"
+  cmdline: "console=ttyS0 page_poison=1"
+init:
+  - linuxkit/init:1b8a7e394d2ec2f1fdb4d67645829d1b5bdca037
+  - linuxkit/runc:3a4e6cbf15470f62501b019b55e1caac5ee7689f
+  - linuxkit/containerd:e3832e460e0565d94141a59bc750e6d6523b271f
```

Comment:
should this bump to `linuxkit/containerd:b50181bc6e0084e5fcd6b6ad3cf433c4f66cae5a`?